tests/queries/0_stateless/01947_multiple_pipe_read.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+SAMPLE_FILE="$CURDIR/01947_sample_data.csv"
```

Comment:
Because it uses one file, it is not possible to run flaky check (runs tests in parallel multiple times) - https://clickhouse-test-reports.s3.yandex.net/25960/bab7681bc18fbaf7df49737ca72cfe70756a5f62/functional_stateless_tests_flaky_check_(address)/test_run.txt.out.log
To fix this, you can simply use `mktemp`, i.e. `mktemp --tmpdir "$CURDIR" 01947_multiple_pipe_read_saple_data_XXXXXX.sh` (completely untested, but you can check it using `clickhouse-test --test-runs 100 01947_multiple_pipe_read. --jobs 8`)